src/main/java/org/broadinstitute/hellbender/tools/validation/CompareBaseQualities.java

```diff
@@ -30,8 +33,28 @@
     @Argument(doc="throw error on diff", shortName = "cd", fullName = "throwOnDiff", optional = true)
     protected boolean throwOnDiff = false;
 
+    /**
+     * Use static quantized quality scores to a given number of levels.
```

Comment:
A bunch of duplicated code  between these tools now.  Should the arguments be extracted out to an argument collection?